esmvalcore/preprocessor/_area.py

```diff
@@ -331,14 +331,21 @@ def _crop_cube(cube, start_longitude, start_latitude, end_longitude,
         lon_bound = lon_coord.core_bounds()[0]
         lon_step = lon_bound[1] - lon_bound[0]
         start_longitude -= lon_step
+        if start_longitude < -180:
+            start_longitude = -180.
```

Comment:
```suggestion
        if start_longitude < 0:
            start_longitude = 0.
```

Reply:
Could you also change it here?

Reply:
done and adjusted tests accordingly as well :beer: